analyticsclient/course.py

```diff
@@ -105,6 +105,16 @@ def problems(self, data_format=DF.JSON):
         path = 'courses/{0}/problems/'.format(self.course_id)
         return self.client.get(path, data_format=data_format)
 
+    def problems_and_tags(self, data_format=DF.JSON):
+        """
+        Get the problems for the course with assigned tags.
+
+        Arguments:
+            data_format (str): Format in which data should be returned
```

Comment:
I'm totally new to this API: is it obvious what formats are allowed?  Can this docstring elaborate?